index.html

```diff
@@ -1,20 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <title>Dwyl Application</title>
-        <script src="./elm.js"></script>
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta name="Description" content="DWYL application, built with Elm!">
-        <link rel="canonical" href="https://app-mvp-elm.herokuapp.com"/>
-        <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
-    </head>
 
-    <body>
-        <noscript>
-            <span>This website is using Elm, so it needs js enabled</span>
-        </noscript>        
-       <script>
-            Elm.Main.init();
-        </script>
-    </body>
+<head>
+    <title>Dwyl Application</title>
+    <script src="./elm.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="Description" content="DWYL application, built with Elm!">
+    <link rel="canonical" href="https://app-mvp-elm.herokuapp.com" />
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css" />
+</head>
+
+<body>
+    <noscript>
+        <span>This website is using Elm, so it needs js enabled</span>
+    </noscript>
+    <script>
+        var flags = localStorage.getItem('store'); // get the person session from the local storage
+        var app = Elm.Main.init({ flags: flags });
```

Comment:
does the app override the whole `<body>` element?

Reply:
yes. When you initialise a `Elm.application` Elm will manage the html body for you (and the title of the page)
From: https://guide.elm-lang.org/webapps/

> You load whatever you need in the <head> and you initialize your Elm program in the <body>. The Elm program will take it from there and render everything.